common/metrics/defs.go

```diff
@@ -2161,9 +2172,9 @@ var MetricDefs = map[ServiceIdx]map[int]metricDefinition{
 		ServicePendingRequests:                              {metricName: "service_pending_requests", metricType: Gauge},
 		ServiceFailures:                                     {metricName: "service_errors", metricType: Counter},
 		ServiceCriticalFailures:                             {metricName: "service_errors_critical", metricType: Counter},
-		ServiceLatency:                                      {metricName: "service_latency", metricType: Timer},
-		ServiceLatencyNoUserLatency:                         {metricName: "service_latency_nouserlatency", metricType: Timer},
-		ServiceLatencyUserLatency:                           {metricName: "service_latency_userlatency", metricType: Timer},
+		ServiceLatency:                                      {metricName: "service_latency", metricType: Timer, unit: Milliseconds},
```

Comment:
Having to specify unit for Timer feels wrong, it should default to miliseconds for timer. Maybe define method and use NewTimerMetric("service_latency") here which sets type to Timer and unit to miliseconds.

Reply:
I leave this option in case if we want to have different histogram bucket sets for different timers. Say one for long and one for short operations.